blocks/note/note.css

```diff
@@ -25,7 +25,7 @@
   margin: 20px 0;
   padding: 20px;
   font-size: 16px;
-  max-width: var(--grid-container-width);
+  max-width: var(--grid-container-width-helpx);
```

Comment:
Instead of -helpx can we call it like helparticle or threecol

Reply:
@adeshmukh-1585 I changed the name to `...-help-article`